libs/vm/src/analyser.cpp

```diff
@@ -1954,6 +2112,14 @@ FunctionPtr Analyser::FindFunction(TypePtr const &type, FunctionGroupPtr const &
   if (functions.size() == 1)
   {
     actual_types = array[0];
+    for (std::size_t i{}; i < actual_types.size(); ++i)
+    {
+      auto const &supplied_node = supplied_nodes[i];
+      if (supplied_node->node_kind == NodeKind::InitializerList)
+      {
+        ConvertInitializerList(supplied_node, actual_types[i]);
+      }
+    }
```

Comment:
+1 nice solution to this issue, convert the nodes as soon as we know the winning signature.

Reply:
Sure. :)